api/envoy/extensions/access_loggers/filters/local_ratelimit/v3/local_ratelimit.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.extensions.access_loggers.filters.local_ratelimit.v3;
+
+import "envoy/config/core/v3/config_source.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.access_loggers.filters.local_ratelimit.v3";
+option java_outer_classname = "LocalRatelimitProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/access_loggers/filters/local_ratelimit/v3;local_ratelimitv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: LocalRateLimitFilter]
+// [#extension: envoy.access_loggers.extension_filters.local_ratelimit]
+
+// Filters for rate limiting the access log emission using global token buckets per process and shared across all listeners.
+message LocalRateLimitFilter {
```

Comment:
I would argue that the following is not a rate-limiting filter, but a dynamic one.
The most basic one will be a filter that just has a TokenBucket configured.
The dynamic fetching will be another approach:
1. define an xDS type:
```
message TokenBucketConfig {
  string name = 1;
  ConfigSource config_source = 2;
}
```
3. add a field to the LocalRateLimitFilter of the TokenBucketConfig so it can be configured to fetch the buckets dynamically.

Alternatively, the filter can be renamed to DynamicTokenBucketFetcher that will implement the current interface.

Reply:
> The most basic one will be a filter that just has a TokenBucket configured.

The main reasons we do in the current way are 1) have per process rate limiting and 2) sharing the config. The dynamicTokenBucket is a side benefit. 

Could you be more specific if you are making some proposal? Thx!

Reply:
Updated.

Reply:
Updated.